QGL/Compiler.py

```diff
@@ -627,3 +631,15 @@ def set_log_level(loggerName='QGL.Compiler', levelDesired=logging.DEBUG,
     logger.addHandler(handler)
     # Without this when running in notebook, console gets log stmts at default format
     logger.propagate = 0
+
+def save_code(seqs, filename):
+    from IPython.lib.pretty import pretty
+    import io
```

Comment:
@blakejohnson  why the `io` import? I dropped it in 19a6bf45599e030f5f48def3293048a257227fb9 and everything still seems to work.